xml/systemd.xml

```diff
@@ -83,19 +80,21 @@
      </para>
     </listitem>
    </itemizedlist>
-  </sect2>
-
+   
   <sect2 xml:id="sec-boot-systemd-unitfile">
    <title>Unit file</title>
    <para>
     A unit configuration file contains information about a service, a socket, a
     device, a mount point, an automount point, a swap file or partition, a
     start-up target, a watched file system path, a timer controlled and
-    supervised by systemd, a temporary system state snapshot, a resource
-    management slice or a group of externally created processes. <quote>Unit
-    file</quote> is a generic term used by systemd for the following:
+    supervised by &systemd;, a temporary system state snapshot, a resource
+    management slice or a group of externally created processes. :
    </para>
-   <itemizedlist mark="bullet" spacing="normal">
+   <para>
+    <quote>Unit file</quote> is a generic term used by &systemd; for the 
+    following:
+   </para>
+    <itemizedlist mark="bullet" spacing="normal">
     <listitem>
      <formalpara>
       <title>Service</title>
```

Comment:
There is a itemized list with a single item? Perhaps replacing this with a single sentence would be better? For example: _ In &systemd; terminology, unit files refer to services_

Reply:
there are more list items following